tools/ci/cirrus_general.yml

```diff
@@ -1,3 +1,5 @@
+# Copyright 2023 Arm Limited and/or its affiliates <open-source-office@arm.com>
+
```

Comment:
Why are you adding a copyright to this file?

Reply:
For the small amount of additional logic changed to pull the submodule.

Reply:
I understand that legal teams like copyright notices, but we don't have a habit of adding copyright notices to files and at least right now I would prefer to keep it that way and you explain this to the legal team at ARM.

If we really did some of these files would be littered by personal or company copyrights (quansight, nvidia, intel, apple) and probably even more universities...

If the legal team needs a more definite no from us, I suspect we can just give that.

Reply:
@seberg I've raised this with our team and I'll let you know the outcome, I have to try and follow best practices first but it's relatively common for us to have a reasonable discussion as to what suits the project 😸 

I would point out that there are a number of explicit copyright claims through-out the codebase which imply this is a practice, maybe worth consideration of how we make that more consistent?